src/compiler/utilities.ts

```diff
@@ -1818,10 +1818,8 @@ namespace ts {
 
         function getJSDocCommentsAndTagsWorker(node: Node): void {
             const parent = node.parent;
-            if (parent &&
-                (parent.kind === SyntaxKind.PropertyAssignment ||
-                 parent.kind === SyntaxKind.PropertyDeclaration ||
-                 getNestedModuleDeclaration(parent))) {
+            if (!parent) return;

```

Comment:
why doesn't this miss a lot of tags, for example, when `node.kind === SyntaxKind.Parameter` on line 1848 and following?